starter/src/main/java/com/starter/fullstack/rest/InventoryController.java

```diff
@@ -33,5 +35,9 @@ public InventoryController(InventoryDAO inventoryDAO) {
   public List<Inventory> findInventories() {
     return this.inventoryDAO.findAll();
   }
-}
 
+  @PostMapping
+  public Inventory createInventory(@RequestBody Inventory inventory) {
```

Comment:
Add a valid annotation to the `inventory` parameter to ensure a correct inventory is saved.
A correct inventory means all of its required fields are set.
See `ProductController.java` for usage of the valid annotation.
There may be changes to the test (i.e., another field needs to be set).